src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -581,10 +581,10 @@ public override bool IsCollapsed
             get => base.IsCollapsed;
             set
             {
-                foreach (var connector in WorkspaceViewModel.GetViewModelsInternal(NodeModel.AllConnectors.Select(x => x.GUID)))
-                {
-                    connector.IsCollapsed = value;
-                }
+                //foreach (var connector in WorkspaceViewModel.GetViewModelsInternal(NodeModel.AllConnectors.Select(x => x.GUID)))
```

Comment:
Delete?

Reply:
hm, this is already deleted. Did you maybe look at one of the commits instead of `Files changed`?

Reply:
Yeah will look at files changed from now on..